spark-cassandra-connector/src/main/scala/com/datastax/spark/connector/cql/CassandraConnectionFactory.scala

```diff
@@ -77,9 +77,33 @@ object DefaultConnectionFactory extends CassandraConnectionFactory {
         } finally {
           IOUtils.closeQuietly(trustStoreFile)
         }
-
+        val kmf = if(conf.clientAuthEnabled){
+            conf.keyStorePath map {
+              case path ⇒
```

Comment:
please don't use this special characters, you can achieve nice looking arrows by ligatures